From 0298ba9eb58f266c63566a872221e42366fcd079 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Fri, 5 Sep 2014 18:56:13 -0400
Subject: [redhat] configs: Enable config options for bzImage signature verification

Message-id: <1409943373-12669-39-git-send-email-vgoyal@redhat.com>
Patchwork-id: 91520
O-Subject: [RHEL7.1 PATCH 38/38] config: Enable config options for bzImage signature verification
Bugzilla: 852066
RH-Acked-by: Dave Anderson <anderson@redhat.com>
RH-Acked-by: Baoquan He <bhe@redhat.com>
RH-Acked-by: Dave Young <dyoung@redhat.com>
RH-Acked-by: Chao WANG <chaowang@redhat.com>

BZ:852066

Well, all the patches are in. Enable right config options to enable
new kexec syscall and bzImage signature verificaiton.

Signed-off-by: Vivek Goyal <vgoyal@redhat.com>
---
 create mode 100644 redhat/configs/generic/x86_64/CONFIG_KEXEC_BZIMAGE_VERIFY_SIG
 create mode 100644 redhat/configs/generic/x86_64/CONFIG_KEXEC_FILE
 create mode 100644 redhat/configs/generic/x86_64/CONFIG_KEXEC_VERIFY_SIG

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_KEXEC_BZIMAGE_VERIFY_SIG b/redhat/configs/generic/x86_64/CONFIG_KEXEC_BZIMAGE_VERIFY_SIG
new file mode 100644
index 0000000..e740740
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_KEXEC_BZIMAGE_VERIFY_SIG
@@ -0,0 +1 @@
+CONFIG_KEXEC_BZIMAGE_VERIFY_SIG=y
diff --git a/redhat/configs/generic/x86_64/CONFIG_KEXEC_FILE b/redhat/configs/generic/x86_64/CONFIG_KEXEC_FILE
new file mode 100644
index 0000000..25862bf
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_KEXEC_FILE
@@ -0,0 +1 @@
+CONFIG_KEXEC_FILE=y
diff --git a/redhat/configs/generic/x86_64/CONFIG_KEXEC_VERIFY_SIG b/redhat/configs/generic/x86_64/CONFIG_KEXEC_VERIFY_SIG
new file mode 100644
index 0000000..5f39f19
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_KEXEC_VERIFY_SIG
@@ -0,0 +1 @@
+CONFIG_KEXEC_VERIFY_SIG=y
-- 
1.7.1